README.md

```diff
@@ -0,0 +1,150 @@
+# Systems Research Artifacts (sysartifacts)
+
+Sysartifacts (https://sysartifacts-generalize.github.io/) hosts the artifact

```

Comment:
Should we make this link point at `https://sysartifacts.github.io/` (rather than `-generalize`)? I would suggest we take down the `-generalize` site after merging this, otherwise we might end up with some people pointing their links at the `-generalize` site, which was only intended to be a staging area for testing out the new templating setup.

Reply:
Agreed -- I will delete the `sysartifacts-generalize` as soon we merge.
In this specific case, a `(/)` link might work for both cases.

Reply:
Made this change in 3812b33b49c4b0c6198f329ff41d3bafed2bdfd3. I think it's better to include an absolute URL, since this README will get rendered on the GitHub repository page, not the `sysartifacts.github.io` page.

Reply:
Ah right!